### ISSUE TITLE
The attribute aria-expanded is not supported by the role menuitem.

### ISSUE THREAD

**Issue by jlurena** [jsx-eslint/eslint-plugin-jsx-a11y#630]
A menu item, acting as a submenu given a role of `menuitem` should have an `aria-expanded` attribute. I am being thrown an error when I do this.

Something like this raises an error:
```
<ul role="menubar">
  <li>
    <a role="menuitem" aria-expanded="false" href="someurl.com">Some URL</a>
      <ul role="menu">
        <li><a href="https://google.com">Google.com</a></li>
        <li><a href="https://github.com">Github.com</a></li>
      </ul>
  </li>
</ul>
```

> The attribute aria-expanded is not supported by the role menuitem.

According to https://github.com/w3c/aria/issues/454 this is valid aria spec.

This is also shown on an example on W2 
https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html

**Comment by Semigradsky** [comment]
Any progress here?

**Comment by jessebeach** [comment]
This error does not occur in v6.3.x.

<img width="528" alt="jessebeach_jessebeach-mbp____code_evcohen_eslint-plugin-jsx-a11y" src="https://user-images.githubusercontent.com/345913/85213443-46b15a00-b313-11ea-8035-43e9e25c85a0.png">
